.circleci/config.yml

```diff
@@ -101,6 +101,14 @@ jobs:
             - packages/instantsearch.js/es
             - packages/instantsearch.js/cjs
             - packages/instantsearch.js/dist
+            - packages/react-instantsearch/dist
```

Comment:
Unrelated to this PR, but `yarn run build --ignore='example-*'` in l. 91 could be changed into `yarn run build:ci` just like we have a `test:ci` script.

It's easier to follow when you're trying to debug CI scripts locally.